web-ui/src/components/menu/Menu.jsx

```diff
@@ -9,9 +9,9 @@ import {
   selectHasAnniversaryReportPermission,
   selectHasBirthdayReportPermission,
   selectHasCheckinsReportPermission,
-  selectHasPulseReportPermission,
```

Comment:
This was renamed to `selectHasViewPulseReportPermission` for consistency with other permissions.